app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: "str", appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.alive.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
+            return 0
```

Comment:
You have an extra `else` statement in your `feed` function. According to the checklist, you can simply return 0 without the `else` keyword. This will make your code cleaner. Here is how you can do it:
python
if self.is_hungry:
    print(f"Eating {self.appetite} food points...")
    self.is_hungry = False
    return self.appetite
return 0
```